tdms/src/iterator.cpp

```diff
@@ -5878,12 +5622,6 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[]) {
   free(Dzu);
   //  free(lower_boundary_update);
   free(Nt);
-  free(I0);
-  free(I1);
-  free(J0);
-  free(J1);
-  free(K0);
-  free(K1);

```

Comment:
Automatically/implicitly in the InterfaceComponent destructor? I don't see the explicit `free` ing.

Reply:
yep – assumed we're all good with an implicit trivial destructor if the members are only an int and bool(??)